core/assets/bundles/bundle_nl.properties

```diff
@@ -118,91 +118,91 @@ players = {0} Spelers online
-confirmunban = Are you sure you want to unban this player?
-confirmadmin = Are you sure you want to make this player an admin?
-confirmunadmin = Are you sure you want to remove admin status from this player?
-joingame.title = Join Game
-joingame.ip = Address:
-disconnect = Disconnected.
-disconnect.error = Connection error.
-disconnect.closed = Connection closed.
+confirmban = Weet je zeker dat je deze speler wilt verbannen?
+confirmkick = Weet je zeker dat je deze speler wilt kicken?
+confirmvotekick = Weet je zeker dat je deze speler wilt vote-kicken?
+confirmunban = Weet je zeker dat je deze speler niet meer wilt verbannen?
+confirmadmin = Weet je zeker dat je deze speler een admin wil maken?
+confirmunadmin = Weeet je zeker dat je deze speler niet meer admin wilt laten zijn?
+joingame.title = Spel toetreden
+joingame.ip = Adres:
+disconnect = Verbinding verbroken.
+disconnect.error = Verbindingsfout.
+disconnect.closed = Verbinding gesloten.
 disconnect.timeout = Timed out.
-disconnect.data = Failed to load world data!
-cantconnect = Unable to join game ([accent]{0}[]).
-connecting = [accent]Connecting...
-connecting.data = [accent]Loading world data...
+disconnect.data = Laden van speldata mislukt!
+cantconnect = Niet mogelijk het spel toe te treden ([accent]{0}[]).
+connecting = [accent]Verbinden...
+connecting.data = [accent]Speldata laden...
 server.port = Port:
-server.addressinuse = Address already in use!
-server.invalidport = Invalid port number!
-server.error = [crimson]Error hosting server: [accent]{0}
-save.new = New Save
-save.overwrite = Are you sure you want to overwrite\nthis save slot?
-overwrite = Overwrite
-save.none = No saves found!
-saveload = [accent]Saving...
-savefail = Failed to save game!
-save.delete.confirm = Are you sure you want to delete this save?
-save.delete = Delete
-save.export = Export Save
-save.import.invalid = [accent]This save is invalid!
+server.addressinuse = Adres al in gebruik!
+server.invalidport = Ongeldig port nummer!
+server.error = [crimson]Fout in host server: [accent]{0}
+save.new = Nieuw opgeslagen spel
+save.overwrite = Weet je zeker dat je dit opgeslagen\nspel wilt overschrijven?
+overwrite = Overschrijven
+save.none = Geen opgeslagen spellen gevonden!
+saveload = [accent]Opslaan...
+savefail = Spel opslaan mislukt!
+save.delete.confirm = Weet je zeker dat je dit opgeslagen spel wilt verwijderen?
+save.delete = Verwijderen
+save.export = Export opgeslagen spel
+save.import.invalid = [accent]Dit opgeslagen spel is ongeldig!
 save.import.fail = [crimson]Failed to import save: [accent]{0}
 save.export.fail = [crimson]Failed to export save: [accent]{0}
 save.import = Import Save
 save.newslot = Save name:
-save.rename = Rename
-save.rename.text = New name:
-selectslot = Select a save.
+save.rename = Hernoemen
+save.rename.text = Nieuwe naam:
+selectslot = Selecteer een opgeslagen spel.
 slot = [accent]Slot {0}
-editmessage = Edit Message
+editmessage = Bericht bijwerken
```

Comment:
I wouldn't use "bijwerken" in this context since this would imply the message to be outdated. "bewerken" would be a better solution to get rid of this connotation.

Reply:
I've probably just thought too quickly. Thnx for pointing out all the errors!